be/src/Project/Services/Identity/Identity.Api/Services/Users/UsersService.cs

```diff
@@ -30,13 +32,15 @@ public class UsersService : IUsersService
         private readonly IStringLocalizer accountLocalizer;
         private readonly IUserService userService;
         private readonly LinkGenerator linkGenerator;
+        private readonly IHttpContextAccessor httpContextAccessor;
```

Comment:
I used this to get full url link, because when i used function getPathByAction i receveid example:  /pl/Accounts/SetPassword/Index/1231-1231-123-123. 

I didn't want to use variables which contains domain.
What you think?